config/_default/languages.toml

```
@@ -0,0 +1,28 @@
+[en]
+  languageName = "English"
+  contentDir = "content/en"
```

```suggestion
  contentDir = "content/"
```
Can we keep the directory structure for the content the same?
That would make this PR much smaller and let's us keep the file history for the actual docs.

Basically reverting changes like this:
[content/concepts/fundamentals/addressing.md → content/en/docs/fundamentals/addressing.md](https://github.com/libp2p/docs/pull/220/files#diff-e69ee054730b149e391f572e7b10dce989535ff450796f0c7aa1135b2a425f7e)